src/dvclive/live.py

```diff
@@ -111,22 +112,7 @@ def _init_resume(self):
         logger.debug(f"{self._step=}")
 
     def _init_cleanup(self):
-        for plot_type in PLOT_TYPES:
-            shutil.rmtree(
-                Path(self.plots_dir) / plot_type.subfolder, ignore_errors=True
-            )
-
-        for f in (
-            self.metrics_file,
-            self.params_file,
-            os.path.join(self.dir, "report.html"),
-            os.path.join(self.dir, "report.md"),
-        ):
-            if f and os.path.exists(f):
-                os.remove(f)
-
-        if self.dvc_file and os.path.exists(self.dvc_file):
-            os.remove(self.dvc_file)
+        shutil.rmtree(self.dir)
```

Comment:
If we do this, let's cleanup the entire dvclive dir so that stale `dvclive/dvc.yaml` files get removed. AFAIR, there was not that strong a motivation to selectively cleanup files in the dir, and I think we have established the convention that this dir will be overwritten each run.

Reply:
Okay, I went with globbing for any `dvc.yaml` files inside the dvclive dir and deleting those.